### ISSUE TITLE
Accessibility: Dropdown menus fail to collapse when focus shifts to adjacent top-level items

### ISSUE THREAD

**Issue by trubach** [web-illinois/ilw-header-menu#34]
**Description of the Issue:**
When a user tabs into a top-level menu item and opens its dropdown, then presses Tab again to move to the next top-level item, the initial dropdown remains expanded. 

**Steps to Reproduce:**
- Go to https://builder3.toolkit.illinois.edu/
- Tab into the main navigation menu
- Expand a top-level menu item that contains a dropdown by pressing enter or the down-arrow
- Press Tab until you've to shifted focus to the next adjacent top-level menu item
- Observe that the first dropdown remains open instead of collapsing.

**Screenshots:**
Screenshot of last menu item having focus:
<img width="705" height="425" alt="Image" src="https://github.com/user-attachments/assets/d4a81328-1add-4f19-8a02-5f0b769d50b9" />

After pressing tab, notice the "Start Here" menu remains open and "Information" has focus:
<img width="597" height="415" alt="Image" src="https://github.com/user-attachments/assets/fd399c65-c3ec-49ab-9b6c-3a86602b09a6" />

This issue was reported by Tim Offenstein

**Comment by sydbeth** [comment]
I believe this is not an issue as long as the dropdowns close when a new one is activated. [See the w3c example](https://www.w3.org/) 

The [mega menu](https://github.com/web-illinois/ilw-header-megamenu) also maintains this behavior so this issue will need to be duplicated there if it needs to be updated
